CHANGELOG.md

```diff
@@ -13,11 +13,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - `libcnb`:
   - Changed `Layer` interface from `&self` to `&mut self`. ([#669](https://github.com/heroku/libcnb.rs/pull/669))
+- `libcnb-package`:
```

Comment:
As of #662, we treat `libcnb-package` (and `libcnb-common`) as an internal packages that has no stability guarantees. As such, we don't mention them directly in the changelog, but instead mention any end-user impact in terms of what the user would experience - which in this case is via `libcnb-cargo` and `libcnb-test`.

However, rather than duplicate the entries under both a `libcnb-cargo` bullet and a `libcnb-test` bullet, what we did for the last cross-compile assistance change was to mention it at the top level, like so:
https://github.com/heroku/libcnb.rs/blob/2907c182b4aaf72d90a312e95455e3b51c0399a8/CHANGELOG.md?plain=1#L36

Could you remove the `libcnb-package` bullet and move the entries to the top level?

Reply:
Ah didn't know we made that distinction (I modeled the entries after [this entry](https://github.com/heroku/libcnb.rs/pull/577/commits/fc4f442b5d47f30c85a92b9af32aac9ccab12f02), which seems to have been the last one prior to marking the `libcnb-package` internal :)).

I [moved the entries up](https://github.com/heroku/libcnb.rs/pull/769/commits/d71b6c5c53cdea63953379969b87a2872bc0af77) to the top level (both under the `Changed` header) without the `libcnb-package` bullet.